src/main/java/uk/gov/hmcts/reform/professionalapi/persistence/PrdEnumRepository.java

```diff
@@ -9,5 +10,5 @@
 @Repository
 public interface PrdEnumRepository extends JpaRepository<PrdEnum, UUID> {
 
-
+    List<PrdEnum> findByEnabled(String flag);

```

Comment:
can you make the column non nullable?
the code does not currently handle null values and will break